harness/determined/exec/harness.py

```diff
@@ -82,6 +82,11 @@ def main(chief_ip: Optional[str]) -> int:
         # without horovod, and we can't load the right horovod until we know which Trial class the
         # user implemented.
         trial_class, controller_class = load.get_trial_and_controller_class(env.experiment_config)
+        if info.container_rank == 0:
+            try:
+                analytics.send_analytics("trial_loaded", analytics.get_trial_analytics(trial_class))
+            except Exception as e:
+                logging.debug(f"Cannot sending analytics: {e}")
```

Comment:
Cannot send~ing~ analytics

But also, a suggestion: let's just drop this exception.  I think any user might be slightly alarmed to see it, and I doubt any user would be particularly inclined to fix it.  It's like a message that only we care about, but which only the user would see.

In the worst case, if something went really wrong with segment then all of our users might get extra messages that they can't really do anything about.

Reply:
>In the worst case, if something went really wrong with segment then all of our users might get extra messages that they can't really do anything about.

right. so are other debug-level messages. It doesn't hurt users but they might tell us if anything goes wrong so that we know we can fix it in the next release.